defaults/main.yml

```diff
@@ -27,12 +27,19 @@ kube_scheduler_requests_memory: "128Mi"
 kube_controller_manager_requests_cpu: "200m"
 kube_controller_manager_requests_memory: "128Mi"
 
-kube_scheduler_policy_config_file_src: ""
-kube_scheduler_policy_config_dir: ""
-kube_scheduler_policy_config_file_name: ""
 kube_scheduler_verbosity: "3"
-
 enable_ext_scheduler: False
+kube_scheduler_policy_config_file_name: "scheduler-policy-config-with-extender.json"
+kube_scheduler_policy_config_file_src: "{{playbook_dir}}/files/{{kube_scheduler_policy_config_file_name}}"
```

Comment:
`kube_scheduler_policy_config_file_src` here should point to a dir local to the role.

Reply:
playbook_dir point to the base ansible directory. Pointint it local to role, then we need to copy kube_scheduler_policy_config_file_name inside the role